[0.028s] Invoking command in '/home/zhijie/autoware/build/mcity_msg': CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/home/zhijie/autoware/install/simulator_compatibility_test:/home/zhijie/autoware/install/dbw_ford_input_test:/home/zhijie/autoware/install/ad_api_visualizers:/opt/ros/humble /usr/bin/cmake /home/zhijie/autoware/src/terasim/preview_control/mcity_msg -DCMAKE_INSTALL_PREFIX=/home/zhijie/autoware/install/mcity_msg
[0.051s] -- Found ament_cmake_auto: 1.3.5 (/opt/ros/humble/share/ament_cmake_auto/cmake)
[0.386s] -- Found rosidl_generator_c: 3.1.5 (/opt/ros/humble/share/rosidl_generator_c/cmake)
[0.388s] -- Found rosidl_adapter: 3.1.5 (/opt/ros/humble/share/rosidl_adapter/cmake)
[0.393s] -- Found rosidl_generator_cpp: 3.1.5 (/opt/ros/humble/share/rosidl_generator_cpp/cmake)
[0.430s] -- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
[0.466s] -- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
[0.826s] [31mCMake Error at /opt/ros/humble/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:59 (message):
[0.826s]   execute_process(/usr/bin/python3.10 -m rosidl_adapter --package-name
[0.827s]   mcity_msg --arguments-file
[0.827s]   /home/zhijie/autoware/build/mcity_msg/rosidl_adapter__arguments__mcity_msg.json
[0.827s]   --output-dir /home/zhijie/autoware/build/mcity_msg/rosidl_adapter/mcity_msg
[0.827s]   --output-file
[0.827s]   /home/zhijie/autoware/build/mcity_msg/rosidl_adapter/mcity_msg.idls)
[0.827s]   returned error code 1:
[0.827s] 
[0.827s]   Error processing 'float64[] xVector' of 'mcity_msg/PlanedPath2': ''xVector'
[0.827s]   is an invalid field name.  It should have the pattern
[0.827s]   '^(?!.*__)(?!.*_$)[a-z][a-z0-9_]*$''
[0.827s] 
[0.827s]   Traceback (most recent call last):
[0.827s] 
[0.827s]     File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
[0.827s]       return _run_code(code, main_globals, None,
[0.827s]     File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
[0.827s]       exec(code, run_globals)
[0.828s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__main__.py", line 19, in <module>
[0.828s]       sys.exit(main())
[0.828s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/main.py", line 53, in main
[0.828s]       abs_idl_file = convert_to_idl(
[0.828s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__init__.py", line 19, in convert_to_idl
[0.828s]       return convert_msg_to_idl(
[0.828s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/msg/__init__.py", line 28, in convert_msg_to_idl
[0.828s]       msg = parse_message_string(package_name, input_file.stem, content)
[0.828s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 519, in parse_message_string
[0.828s]       fields.append(Field(
[0.828s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 354, in __init__
[0.828s]       raise NameError(
[0.828s] 
[0.828s]   NameError: 'xVector' is an invalid field name.  It should have the pattern
[0.828s]   '^(?!.*__)(?!.*_$)[a-z][a-z0-9_]*$'
[0.828s] 
[0.828s] Call Stack (most recent call first):
[0.828s]   /opt/ros/humble/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:130 (rosidl_adapt_interfaces)
[0.828s]   CMakeLists.txt:77 (rosidl_generate_interfaces)
[0.828s] 
[0.828s] [0m
[0.828s] -- Configuring incomplete, errors occurred!
[0.828s] See also "/home/zhijie/autoware/build/mcity_msg/CMakeFiles/CMakeOutput.log".
[0.831s] Invoked command in '/home/zhijie/autoware/build/mcity_msg' returned '1': CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/home/zhijie/autoware/install/simulator_compatibility_test:/home/zhijie/autoware/install/dbw_ford_input_test:/home/zhijie/autoware/install/ad_api_visualizers:/opt/ros/humble /usr/bin/cmake /home/zhijie/autoware/src/terasim/preview_control/mcity_msg -DCMAKE_INSTALL_PREFIX=/home/zhijie/autoware/install/mcity_msg
